src/Docker/Configuration/Component.php

```diff
@@ -15,10 +15,29 @@ public function getConfigTreeBuilder()
     {
         $treeBuilder = new TreeBuilder('component');
         $root = $treeBuilder->getRootNode();
-        $definition = $root->children()->arrayNode('definition')->isRequired();
```

Comment:
Proč je tu `->ignoreExtraKeys(false)`?
Zatím se to asi používá se to snad jenom v src\Docker\Component.php ale řekl bych že tu spíš má být
`->ignoreExtraKeys(true)` ne?

Reply:
🤔 jako asi máš pravdu že nic jinýho z toho se vlastně nepoužívá takže to ostatní nastavení smazat můžeme

Reply:
-> https://github.com/keboola/docker-bundle/pull/750/commits/d70884552476fd802315fa347917ace29dd14202